src/index.html

```diff
@@ -8,6 +8,54 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+      <div class="stars--0 stars">
+        <div class="stars__star none_active"> </div>
+        <div class="stars__star none_active"> </div>
+        <div class="stars__star none_active"> </div>
+        <div class="stars__star none_active"> </div>
+        <div class="stars__star none_active"> </div>
```

Comment:
Don't add additional class names to the `stars__star`. It's redundant. You can use a modifier for the stars block and  :nth() child for it
```suggestion
        <div class="stars__star"> </div>
        <div class="stars__star"> </div>
        <div class="stars__star"> </div>
        <div class="stars__star"> </div>
        <div class="stars__star"> </div>
```